integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -654,6 +655,15 @@ public int unlink(String path) {
   }
 
   /**
+   * Alluxio does not have access time, and the file is created only once. So this operation is a
```

Comment:
file metadata such as permissions can still be modified - would it make sense to use the Alluxio `lastModificationTimeMs` for this?

Reply:
We do update modification time for certain attributes like TTL, pin etc, but none of those is in POSIX API. For permission and ownership, we do not change the modification time for now. And even if we change the time, it should be part of the master implementation.
In other words, we do not expose an API to set the modification time explicitly, and I'm not certain we should expose it. What do you think?

Reply:
sgtm